### PR TITLE
Updated Version

### PR DESCRIPTION
**Pull request by devamop69**

### CODE REVIEW

File: docs/NoSSL/lavalink-without-ssl.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ description: Without SSL should work out of the box!
 ### Hosted by @ [DevamOP](https://bit.ly/m/devam)
 [Discord Server](https://dsc.gg/devam)
 
-Version 3.7.10
+Lavalink v3
```
Comment (DarrenOfficial) [inline docs/NoSSL/lavalink-without-ssl.md:23]: v3?

File: docs/SSL/lavalink-with-ssl.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ description: SSL Uses Secure WS connection, whilst No SSL uses standard WS. if y
 ### Hosted by @ [DevamOP](https://bit.ly/m/devam)
 [Discord Server](https://dsc.gg./devam)
 
-Version 3.7.10
+Lavalink v3
```
Comment (DarrenOfficial) [inline docs/SSL/lavalink-with-ssl.md:21]: v3?

File: docs/NoSSL/lavalink-without-ssl.md (hunk 1 above)
Comment (devamop69) [reply docs/NoSSL/lavalink-without-ssl.md:23]: v3.7.11

File: docs/SSL/lavalink-with-ssl.md (hunk 2 above)
Comment (devamop69) [reply docs/SSL/lavalink-with-ssl.md:21]: v3.7.11

Comment (DarrenOfficial) [reply docs/SSL/lavalink-with-ssl.md:21]: Please fix commit

**Pull request closed without merging**